CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera/CameraViewController+Popups.swift

```diff
@@ -13,36 +13,40 @@ import UIKit
 extension CameraViewController {
 
     @objc func showImportFileSheet() {
-        let alertViewController = UIAlertController(title: nil, message: nil, preferredStyle: .actionSheet)
-        let alertViewControllerMessage: String = NSLocalizedStringPreferredFormat(
-            "ginicapture.camera.popupTitleImportPDForPhotos",
-            comment: "Info label")
+        let alertMessage = NSLocalizedStringPreferredFormat("ginicapture.camera.popupTitleImportPDForPhotos",
+                                                            comment: "Info label")
+        let message = alertMessage.isEmpty ? nil : alertMessage
+        let alertViewController = UIAlertController(title: nil,
+                                                    message: message,
+                                                    preferredStyle: .actionSheet)
+
         if giniConfiguration.fileImportSupportedTypes == .pdf_and_images {
-            alertViewController.addAction(UIAlertAction(title: NSLocalizedStringPreferredFormat(
-                "ginicapture.camera.popupOptionPhotos",
-                comment: "Photos action"),
-                                                        style: .default) { [unowned self] _ in
+            let photosAlertActionTitle = NSLocalizedStringPreferredFormat("ginicapture.camera.popupOptionPhotos",
+                                                                          comment: "Photos action")
+            let photosAlertAction = UIAlertAction(title: photosAlertActionTitle,
+                                                  style: .default) { [unowned self] _ in
+                AnalyticsManager.track(event: .uploadPhotosTapped, screenName: .camera)
                 self.delegate?.camera(self, didSelect: .gallery)
-            })
+            }
+            alertViewController.addAction(photosAlertAction)
         }
 
         alertViewController.view.tintColor = .GiniCapture.accent1
-
-        alertViewController.addAction(UIAlertAction(title: NSLocalizedStringPreferredFormat(
-            "ginicapture.camera.popupOptionFiles",
-            comment: "files action"),
-                                                    style: .default) { [unowned self] _ in
+        let filesAlertActionTitle = NSLocalizedStringPreferredFormat("ginicapture.camera.popupOptionFiles",
+                                                                     comment: "files action")
+        let filesAlertAction = UIAlertAction(title: filesAlertActionTitle,
+                                             style: .default) { [unowned self] _ in
+            AnalyticsManager.track(event: .uploadDocumentsTapped, screenName: .camera)
```

Comment:
Clarification]: shall we first select the flies and then track the event?
If something goes wrong with analytics we might get the UI freeze for the user. 🤔